internal/babel-plugin-lint-no-undefined-identifier/test/fixtures/throws/no-reassignment/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": ["../../../../index"],
```

Comment:
for the relative paths to work here, i had to add another layer of nesting (`/throw` here) for `optionsDir` to be picked up properly. possible `babel-helper-fixtures` bug?